django-stubs/contrib/gis/admin/options.pyi

```diff
@@ -36,7 +37,7 @@ class GeoModelAdmin(ModelAdmin):
     widget: Any = ...
     @property
     def media(self) -> Any: ...
-    def formfield_for_dbfield(self, db_field: Any, request: Any, **kwargs: Any) -> Any: ...
+    def formfield_for_dbfield(self, db_field: Any, request: HttpRequest, **kwargs: Any) -> Any: ...
```

Comment:
we already had this type in the superclass: https://github.com/typeddjango/django-stubs/blob/44806740da9f71de325c1e50586ab6b21fd2c318/django-stubs/contrib/admin/options.pyi#L117

Reply:
Maybe we can remove it from the subtype then?

Reply:
You're right, removed.